.github/workflows/master.yml

```
@@ -220,6 +220,10 @@ jobs:
       - uses: ./.github/actions/setup-python
       - uses: ./.github/actions/setup-pyenv
       - uses: ./.github/actions/setup-java
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: "latest"
```

```suggestion
          version: "x.y.z"
```

Let's pin the version.